plot-lib/plot/private/common/contract.rkt

```diff
@@ -34,8 +34,14 @@
 (define anchor/c (one-of/c 'top-left    'top    'top-right
                            'left        'center 'right
                            'bottom-left 'bottom 'bottom-right))
-(define legend-anchor/c (or/c #f anchor/c
-                              (list/c (one-of/c 'inside 'outside) anchor/c)))
+(define legend-anchor/c (or/c anchor/c
+                              (one-of/c
+                               'no-legend
+                               'outside-top-center-dc
```

Comment:
This anchor position needs a better name.  The term `dc` will not mean anything to a user of the plot package.  Maybe `outside-top-center-plot`, but I am not sure? 

Also, given all the refactoring that was done, I think it will be relatively easy to add new anchor locations in the future.

Reply:
maybe `outside-global-top-center`?

Reply:
sounds good.